backend/remote-state/http/client.go

```diff
@@ -0,0 +1,298 @@
+package http
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	multierror "github.com/hashicorp/go-multierror"
+
+	"github.com/hashicorp/terraform/state"
+	"github.com/hashicorp/terraform/state/remote"
+)
+
+// RemoteClient is used by "state/remote".State to read and write
+// blobs representing state.
+// Implements "state/remote".ClientLocker
+type RemoteClient struct {
```

Comment:
You should comment the attributes of this struct. 

`// The fields below are set from configure`

This file doesn't have a configure function... wondering if this is copy paste error?

Reply:
yeah....beauty of copy paste :)